OJP/OJP_Common.xsd

```diff
@@ -82,6 +82,11 @@
 			<xs:documentation>Reference to an Operator ([TMv6] company  providing public transport services.) </xs:documentation>
 		</xs:annotation>
 	</xs:element>
+	<xs:complexType name="OperatorsRefStructure">
+		<xs:sequence>
+			<xs:element name="OperatorRef" type="siri:OperatorRefStructure" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
This still has `minOccurs="0"` allowing the list to be empty. If the `OperatorsRefStructure` is provided, it should not be empty. Remove `minOccurs="0"`

Reply:
you are right. done